packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -1254,6 +1255,55 @@ const POSTOmnichannelContactsSchema = {
 
 export const isPOSTOmnichannelContactsProps = ajv.compile<POSTOmnichannelContactsProps>(POSTOmnichannelContactsSchema);
 
+type POSTUpdateOmnichannelContactsProps = {
+	contactId: string;
+	name?: string;
+	emails?: string[];
+	phones?: string[];
+	customFields?: Record<string, unknown>;
+	contactManager?: string;
+};
+
+const POSTUpdateOmnichannelContactsSchema = {
+	type: 'object',
+	properties: {
+		contactId: {
+			type: 'string',
+		},
+		name: {
+			type: 'string',
+		},
```

Comment:
Are these supposed to be required or nullable? 🤔

Reply:
The only required property is `contactId`; the other ones are optional and nullable, except for the `name` that cannot accept null as a value. https://ajv.js.org/json-schema.html#nullable